microsoft-edge/devtools-guide-chromium/whats-new/2022/09/devtools-106.md

```diff
@@ -0,0 +1,188 @@
+---
+title: What's new in DevTools (Microsoft Edge 106)
+description: "Introducing the Command Palette. Filter and automatically fix code issues in DevTools for Visual Studio Code. DevTools now uses less disk space when caching sourcemaps. Chromium color themes have been deprecated. Accessibility improvements in Focus Mode and high contrast mode. And more."
+author: MSEdgeTeam
+ms.author: msedgedevrel
+ms.topic: conceptual
+ms.prod: microsoft-edge
+ms.date: 09/29/2022
+---
+# What's New in DevTools (Microsoft Edge 106)
+
+[!INCLUDE [Microsoft Edge team note for top of What's New](../../includes/edge-whats-new-note.md)]
+
+
+<!-- ====================================================================== -->
+## Introducing the Command Palette
+
+<!-- Subtitle: Your favorite commands in one place. -->
+<!--           Your favorite browser and DevTools commands in one place. -->
+
+Command Palette makes it possible to access browser management and developer tools commands with just one keyboard shortcut: `Ctrl`+`Shift`+`Spacebar`.  To access the available browser management commands and DevTools commands, press `Ctrl`+`Shift`+`Spacebar`, and then start typing.  The commands are filtered based on your input characters:
+
+![Microsoft Edge Command Palette](./devtools-106-images/command-palette.png)
+
+To list or filter the available DevTools commands, press `Ctrl`+`Shift`+`Spacebar`, and then enter the greater-than (**>**) character:
+
+![Microsoft Edge Command Palette entering `>` to select DevTools commands](./devtools-106-images/command-palette-devtools.png)
+
+<!--
+*  In Microsoft Edge 106 and most releases of 107, the keyboard shortcut is `Ctrl`+`Shift`+`Spacebar`.
+*  In Microsoft Edge 108 and later, the shortcut is `Ctrl`+`Q`. -->
+The keyboard shortcut is shown on the `edge://flags` page, in the **Command Palette** section:
+
+![The keyboard shortcut for Command Palette in the flags page for version 106](./devtools-106-images/command-palette-flags-shortcut-106.png)
+
+Let us know what commands you'd like to see in future versions of Microsoft Edge, by posting a comment in [Issue 73: [feedback] Command Palette experiment](https://github.com/MicrosoftEdge/DevTools/issues/73) in the **MicrosoftEdge/DevTools** repo!
+
+See also:
+* [Run commands via keyboard with Command Palette](../../../experimental-features/edge-command-palette.md)
+
+
+<!-- ====================================================================== -->
+## Filter and automatically fix code issues in DevTools for Visual Studio Code
+<!-- todo: mention that the extension supports Compatibility and Security Hints? -->
```

Comment:
mention that the extension supports Compatibility and Security Hints?   @codepo8